lib/python/flame/registry/local.py

```
@@ -0,0 +1,124 @@
+                    csv_writer.writerow(["round", metric])
+                    csv_writer.writerow([epoch, metrics[metric]])
+
+    def save_params(self, hyperparameters: Optional[Hyperparameters]) -> None:
+        """Save hyperparameters in a model registry."""
+        with open(os.path.join(self.registry_path, "params", str(self.param_version)), 'wb') as file:
+            pickle.dump(hyperparameters.dict(), file)
+        self.param_version += 1
+
+    def cleanup(self) -> None:
+        """Clean up resources."""
+        pass
+
+    def save_model(self, name: str, model: Any) -> None:
+        """Save a model in a model registry."""
+        model_folder = os.path.join(self.registry_path, "model", name)
+        if not os.path.exists(model_folder):
+            os.mkdir(model_folder)
+        
+        ml_framework = get_ml_framework_in_use()
+        
+        if ml_framework == MLFramework.PYTORCH:
+            import torch
+            # is not safe but matches behavior of MLFlow registry
+            torch.save(model, os.path.join(model_folder, str(self.model_versions[name])))
+        elif ml_framework == MLFramework.TENSORFLOW:
+            model.save(os.path.join(model_folder, str(self.model_versions[name])))
+        self.model_versions[name] += 1
+
+    def load_model(self, name: str, version: int) -> object:
+        """
+        Load a model.
+
+        This method can be called without calling self.setup_run().
+        """
+        ml_framework = get_ml_framework_in_use()
+        model_path = os.path.join(self.registry_path, "model", name, str(version))
+        if ml_framework == MLFramework.PYTORCH:
+            # is not safe but matches behavior of MLFlow registry
+            import torch
+            return torch.load(model_path)
+        elif ml_framework == MLFramework.TENSORFLOW:
+            import tensorflow
+            return tensorflow.keras.models.load_model(model_path)
+        
+        return None
```

how about we raise an error like ModuleNotFoundError, instead of return None?